src/main/resources/org/jenkinsci/plugins/scriptsecurity/scripts/ScriptApproval/tabs/fullScript_approved.properties

```
@@ -0,0 +1,63 @@
+approvedScripts=Approved scripts
+
+# approved scripts
+metadataGatheringDisabled=The metadata gathering was disabled. \
+  <p>You can re-enable it permanently by removing the system property: <br />\
+  <code>org.jenkinsci.plugins.scriptsecurity.scripts.ScriptApproval.metadataGathering</code> </p>\
+  <p>or temporarily (until restart) by running this in the Script Console: <br />\
+  <code>org.jenkinsci.plugins.scriptsecurity.scripts.ScriptApproval.METADATA_GATHERING = true</code></p>
+
+## table headers
+approvedHash=Hash
+approvedHash_tooltip=The hash could be seen as a signature of the script. It's used as an identifier to reference the content of the script.
+approvedExpandCode=Code
+approvedLanguage=Language
+approvedScriptLength=Length
+approvedScriptLength_tooltip=Length of the script in characters
+usageCount=# of uses
+usageCount_tooltip=Usage since the introduction of metadata
+lastTimeUsed=Date of last use
+wasPreapproved_header_tooltip=The presence of a green tick means the script was approved during the configuration. \
+  Otherwise it was created by a user without the permission to run scripts and was then approved using this page.
+lastApprovalTime=Date of the last approval
+lastApprovalTime_tooltip=A single script can be approved multiple times. It occurs every time the related configuration is saved.
+lastKnownApproverLogin=Last approver
+
+## table content
+noRequesterSinceMetadata=N/A
+noRequesterSinceMetadata_tooltip=The requester was not recorded. It could be due to the request being created before the metadata introduction or the code calling this script does not provide the user information.
```

```suggestion
noRequesterSinceMetadata_tooltip=The requester was not recorded. It could be due to the request being created before the metadata introduction or the code calling this script did not provide user information.
```
Note: this line is really long and hard to suggest against w/ GitHub (I accidentally left this page while trying to scroll to comment)